aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md

```diff
@@ -151,19 +113,12 @@ Transformers can be registered onto the document via the `UseTransformer` call o
 * Register a document transformer using a DI-activated `IOpenApiDocumentTransformer`.
 * Register an operation transformer using a delegate.
 
-```csharp
-builder.Services.AddOpenApi(options =>
-{
-    options.UseTransformer((document, context, cancellationToken) => {});
-    options.UseTransformer(new MyDocumentTransformer());
-    options.UseTransformer<MyDocumentTransformer>();
-    options.UseOperationTransformer((operation, context, cancellationToken) => {});
-})
-```
+<!-- PU review replacement code. Original Snippet did not compile. -->
```

Comment:
@mikekistler pls review the code I moved to the project. Original wouldn't compile.